tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-price-availability-hook-test.php

```
@@ -0,0 +1,368 @@
+	 * the result decides which signing secret validates the request.
+	 *
+	 * @dataProvider provide_event_types
+	 * @param string $event_type The event type to classify.
+	 * @param bool   $expected   Whether it is an agentic hook.
+	 */
+	public function test_is_agentic_hook_type( string $event_type, bool $expected ) {
+		$method = new ReflectionMethod( WC_Stripe_Webhook_Handler::class, 'is_agentic_hook_type' );
+		$method->setAccessible( true );
+
+		$this->assertSame( $expected, $method->invoke( $this->handler, $event_type ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_agentic_hook_type is not named in camelCase. (undefined)

(CamelCaseMethodName)